src/macrofier/engine.mjs

```diff
@@ -1113,6 +1131,7 @@ function insertMethodMacros(template, methodObj, json, templates, examples = {})
   template = insertExampleMacros(template, examples[methodObj.name] || [], methodObj, json, templates)
 
   template = template.replace(/\$\{method\.name\}/g, method.name)
+    .replace(/\$\{method\.json\.name\}/g, options.polymorphic ? methodObj.actualName || method.name : method.name)
```

Comment:
1. let's rename the macro to `${method.rpc.name}`
2. I know we're just doing this in memory, but it' like to stick to valid OpenRPC. Can we leave the "actualName" value in `methodObj.name` and create a new `methodObj.title` field for the individual names, e.g. navigateToLaunchIntent.
3. we don't need to pass options around in order to do this... just change the code to:  `methodObj.title || methodObj.name` and it will work for all methods (plus we've created an OpenRPC compatible way to override method names in additional use cases if they come up, since any object can have a title property).

Reply:
Ok, let me try

Reply:
Here we have to use reverse like methodObj.name for individual names, e.g. navigateToLaunchIntent. and methodObj.title to keep actual name. since we are using method.name in all places, we will get new generated name properly for polymorphic case